### ISSUE TITLE
Arabic file transilation 

### ISSUE THREAD

**Issue by Mustafa-Asaad-iq** [electron/i18n#1849]
I want to transelate the documentation to arabic but it's not available so hope you add ar-ir file or  do what you know so i can transelate

**Comment by Ahmedezzzzzzssd** [comment]
do find any soulution

**Comment by MustafaLinux** [comment]
> do find any soulution

NO, they just ignore me... someone told me to message the admins on [Crowdin](https://crowdin.com/projects/electron)
but I just forget about this thing and went to do better stuff, you can message them if you want
